addons/analytic/static/src/components/analytic_distribution/analytic_distribution.js

```diff
@@ -616,7 +616,7 @@ export class AnalyticDistribution extends Component {
 
         const selectors = [
             ".o_popover",
-            ".modal:not(.o_inactive_modal)",
+            ".modal:not(.o_inactive_modal):not(:has(.o_act_window))",
```

Comment:
I think this doesn't work. I may be mistake but this isn't jquery so `:has()` selector can not be used, right ?

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/:has#browser_compatibility
It works on runbot. It might be too recent maybe.

Reply:
OOOH didn't now about that. Noice !